wdl2cwl/main.py

```diff
@@ -394,6 +387,10 @@ def get_memory_requirement(
             if isinstance(memory_runtime, WDL.Expr.String):
                 ram_min_literal = self.get_memory_literal(memory_runtime)
                 return ram_min_literal
+            elif isinstance(memory_runtime, WDL.Expr.Apply):
+                expr, unit = memory_runtime.arguments
```

Comment:
For instances where the memory is concatenated with a function expression to form a string. Like in the case of `memory: select_first([machine_mem_gb, 13]) + " GB"`